usecases/user/open-chat.md

```diff
@@ -67,12 +64,21 @@
         <td>
             <ul>
                 <li>
-                    Item 1
+                    2| Actor is not longer registered to the system
```

Comment:
this exception cannot occur as there is a precondition stating that the user must be logged in